Library/Homebrew/cask/lib/hbc/dsl.rb

```diff
@@ -206,16 +204,6 @@ def sha256(arg = nil)
       @sha256 ||= arg
     end
 
-    def license(arg = nil)
```

Comment:
@jawshooah fixed it by adding `#{HOMEBREW_PREFIX}/Caskroom` into `paths_to_delete`. Please have a look make sure it's okay. thanks!